.gitignore

```diff
@@ -1,2 +1,3 @@
 tags
 target
+.DS_Store
```

Comment:
That should not be here, you can include such OS specific exclude in your global gitignore, ty